content/AppleAuth gem integration with Ruby on Rails - Antonieta Alvarez.md

```diff
@@ -0,0 +1,56 @@
+# AppleAuth gem, Rails Apple Sing In integration
+
+<img src="images/apple_auth.png" alt="Apple" />
+
+## What is AppleAuth?
+
+[AppleAuth](https://rubygems.org/gems/apple_auth) is a rootstrap developed Ruby gem to integrate Apple Sign In in our server side applications. 
+
+## Why implement apple sign in in our projects?
+
+This year, on the last iOS 13 release, "Sign in with Apple" login was one of the new features. At the same time the announce and update his App Store Review Guideline that if your ios app implements a third-party or social login service, like login with Facebook, will be mandatory to offer apple sign in as one option, by the end of April 2020.  Besides following the standards, implement this user authentication will let sign in using their two-factor authentication Apple ID. After the user follows Sign in with Apple to log in, your app receives tokens and user information that you can use to authenticate the user in your server.
```

Comment:
I think the dates are not correct.
```suggestion
Apple announced Sign in with Apple in the (WWDC 2019 Conference)[https://developer.apple.com/videos/play/wwdc2019/706/], then in September 2019 Apple updated his (App Store Review Guideline) [https://developer.apple.com/news/?id=09122019b] to state that if your iOS app implements a third-party or social login service (like Facebook Login or Google Sign-In) it will be mandatory to offer Apple sign in as an option by the end of April 2020.  
```